### PR TITLE
Gigasecond

### LINKED ISSUE #80: exercise Gigasecond

### PR DESCRIPTION
**Pull request by glennj**
Closes #80

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -204,6 +204,15 @@
         "difficulty": 3,
         "status": "active"
       },
+      {
+        "slug": "gigasecond",
+        "name": "Gigasecond",
+        "uuid": "6a6aca53-2dd8-4673-9f72-893eeaadd368",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 4,
```
Comment (IsaacG) [inline config.json:213]: We may need to revisit difficulties.

**Review by IsaacG — APPROVED** [review]

File: config.json (hunk 1 above)
Comment (glennj) [reply config.json:213]: Lower?

Comment (IsaacG) [reply config.json:213]: Eh. It requires reading the time functions. But beyond that it's not too bad. I'd do 3.

Comment (glennj) [reply config.json]: Fair enough

**Pull request merged**